src/neptune_scale/__init__.py

```diff
@@ -366,20 +408,27 @@ def _wait(
 
         begin_time = time.time()
         wait_time = min(sleep_time, timeout) if timeout is not None else sleep_time
-        last_queued_sequence_id = self._operations_queue.last_sequence_id
         last_print_timestamp: Optional[float] = None
 
         while True:
             try:
                 with self._lock:
                     if not self._sync_process.is_alive():
-                        if verbose:
+                        if verbose and not self._is_closing:
```

Comment:
We can probably move the flag check `_is_closing` way higher (even in the `while` condition)

Reply:
The goal here was not to issue a warning when it's expected for the sync process to be killed, ie. during closing.